data_extractor/google_search_history/__init__.py

```diff
@@ -74,11 +75,12 @@ def __extract(data):
     # (i.e., pre/during/after Dutch curfew)
     dates = {'before_news': [], 'during_news': [], 'post_news': [],
              'before_other': [], 'during_other': [], 'post_other': []}
-    earliest = datetime.today()
-    latest = datetime(2000, 1, 1)
+    earliest = datetime.today().astimezone(ZONE)
+    latest = datetime(2000, 1, 1).astimezone(ZONE)
     for data_unit in data["Browser History"]:
         if data_unit["page_transition"].lower() != 'reload':
             time = datetime.fromtimestamp(data_unit["time_usec"]/1e6)
+            time = time.astimezone(ZONE)
             if time < earliest:
                 earliest = time
             if time > latest:
```

Comment:
What happens when `time==earliest`? Make sure you catch all cases, otherwise variables may be undefined.

Reply:
changes earliest and latest calculation using min and max, so issue solved ;)